### ISSUE TITLE
Nodes are not connecting

### ISSUE THREAD

**Issue by doriangorski** [FlowiseAI/Flowise#1754]
**Describe the bug**
Installed locally using node and also using render. Both times, I wasn't able to make connections. Please see the video here - https://www.loom.com/share/4f1b9890d0804048b770067cea98c751?sid=c92918ab-fd96-45b5-8218-e4428dae28b1

**Comment by HenryHengZJ** [comment]
that is strange. any luck creating from scratch?

**Comment by doriangorski** [comment]
Yes, multiple times locally and on Railway, Flowise, HugginF. - same issue

**Comment by JulieAndroid** [comment]
Hi, I've got the same issue, doesn't happen connecting to every node just some. Any solution found for this? Cheers

**Comment by lilahaugust** [comment]
Hi, I'm also having this issue, any luck finding a solution?

**Comment by sapientsnap** [comment]
Connecting output lines intermittent not attaching to inputs.

**Comment by MarioPagliarini** [comment]
This all seems like a scam! Nothing works even with the help of AI! I regret this Flowise! There is no technical support probably because it doesn't work!

**Comment by bkoprivica** [comment]
I can confirm it DOES not work. It is IMPOSSIBLE to connect two nodes.

**Comment by rinaldo77** [comment]
Same here, V2 agent flows basically dont work.
